src/Monitor/Monitor/help/Set-AzActivityLogAlert.md

```diff
@@ -459,4 +459,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [New-AzActionGroup](./New-AzActionGroup.md)
 
-[New-AzActivityLogAlertCondition](./Get-AzActivityLogAlertCondition.md)
+[New-AzActivityLogAlertCondition](./New-AzActivityLogAlertCondition.md)
```

Comment:
Please add an entry to the chagelog for this change, per the PR checklist

Reply:
Have now updated change log, let me know if the format is Ok in changelog.md